maasjuju_toolkit/maas/clone_config.py

```diff
@@ -76,11 +77,11 @@ def main():
     """parses arguments and does work"""
 
     parser = argparse.ArgumentParser(
-        description='Αdd tags to many MaaS machines'
-    )
+        description='Clone MaaS machines configuration')
```

Comment:
Clones*

Reply:
All other scripts use this convention

Reply:
Ok then, I will accept this PR as it is LGTM